packages/vite/src/node/plugins/resolve.ts

```diff
@@ -104,7 +104,7 @@ export function resolvePlugin(baseOptions: InternalResolveOptions): Plugin {
       }
 
       // fast path for commonjs proxy modules
-      if (/\?commonjs/.test(id) || id === 'commonjsHelpers.js') {
+      if (id.includes('commonjs') {
```

Comment:
```suggestion
      if (id.includes('commonjs')) {
```

Reply:
I just realized they are not equivalent? doesn't it need to be `?commonjs`?

Reply:
In your previous commit, you removed it

https://github.com/vitejs/vite/pull/6233/commits/d22c4fb7d17fab294220f6a73d7ac0879d1a3d43#diff-9b81bb364c02eab9494a7d27a5effc400cacbffd3b8f349c192f890c37bfc83fR107

Reply:
welp, my bad